src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php

```diff
@@ -43,24 +43,21 @@ public function __construct(ContainerInterface $container)
      *                             the name of the method that is invoked on listeners.
      * @param EventArgs $eventArgs The event arguments to pass to the event handlers/listeners.
      *                             If not supplied, the single empty EventArgs instance is used.
-     *
-     * @return bool
```

Comment:
Should we use inheritdoc or even just remove phpdocs?

Reply:
inheritdoc I suppose